packages/avatar/tests/avatar-group.test.tsx

```diff
@@ -24,3 +24,29 @@ test("renders a number avatar showing count of truncated avatars", () => {
   const moreLabel = tools.getByText("+3")
   expect(moreLabel).toBeInTheDocument()
 })
+
+test("does not render a number avatar showing count of truncated avatars if max is equal to avatars given", async () => {
+  const tools = render(
+    <AvatarGroup max={5}>
+      <Avatar />
+      <Avatar />
+      <Avatar />
+      <Avatar />
+    </AvatarGroup>,
+  )
+  const moreLabel = tools.queryByTestId("avatar-excess")
+  expect(moreLabel).toBeFalsy()
```

Comment:
Please use `.not.toBeInTheDocument()` instead :) see https://github.com/testing-library/eslint-plugin-testing-library/blob/master/docs/rules/prefer-presence-queries.md